src/interface/src/app/treatments/prescriptions.ts

```diff
@@ -1,6 +1,7 @@
 // placeholder for adding colors based on prescription actions
 
 export type PrescriptionSingleAction =
+  | 'NO_TREATMENT'
```

Comment:
TODO: I'm not sure if this is an explicit PrescriptionAction, or if we should just anticipate a null when we don't have anything here.

Reply:
I dont think we should add it here, we should just get null instead